types/rdfjs__term-map/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for @rdfjs/term-map 1.0
+// Project: https://github.com/rdfjs-base/term-map
+// Definitions by: tpluscode <https://github.com/tpluscode>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Term } from "rdf-js";
+
+declare namespace TermMap {
+    interface TermMap<T extends Term = Term, V = any> extends Map<T, V> {
+    }
+}
+
+interface TermMap<T extends Term = Term, V = any> extends TermMap.TermMap<T, V> {
```

Comment:
Why do you need this interface.. Instead use the class and fix the tslint error

Reply:
This is a feature called [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces). By declaring an interface with same name as the class below and matching type parameters it is possible to avoid declaring all the implemented members.

Reply:
Given that these are all the interfaces in same module i dont see reason why there should be separate interface and class. Please fix the ts lint error by removing the interface here

Reply:
If I remove this interface, I will have to explicitly add all methods of the built-in `Set`. I want to avoid that because it has not value

Reply:
If thats what is the intent then i would recommend removing 
```
declare namespace TermMap {
    interface TermMap<T extends Term = Term, V = any> extends Map<T, V> {
    }
}
```
and keeping single definition

Reply:
Oh goodie, thank you. I tried that before but merging only happens when the class and interface are in same namespace.

I did not think of putting the class **inside** this namespace and export that 👍

Reply:
One question though. When the interface and class are in the same namespace, how are they distinguished when imported as `import { TermMap } from '@rdfjs/term-map'`. This intended to be only the interface but might be wrongly interrupted as class